metric/system/process/process_types.go

```diff
@@ -103,6 +103,10 @@ type ProcLimits struct {
 
 // Implementations
 
+func (t CPUTotal) IsZero() bool {
```

Comment:
Where does this get used? Is there an interface somewhere with this method?

Reply:
Yes. It's used by the `go-structform` code to check to see if a struct should be removed from whatever JSON event is being generated.